### PR TITLE
student comment course

### PR DESCRIPTION
**Pull request by wowKimsohyun17**
## Related Tickets:
- [#30238](https://edu-redmine.sun-asterisk.vn/issues/30238)

## WHAT this PR does?
process student comment course and show comments

## Notes Impacted Areas:
- N/A.

## Attachments:
comment-course

### CODE REVIEW

**Comment by wowKimsohyun17** [comment]
ready

**Comment by TDM1911** [comment]
điền thêm ticket Id vào nhé.

File: app/Http/Controllers/User/StudentController.php [hunk 1]
```diff
@@ -21,4 +25,15 @@ public function showLesson($id)
...
         return view('user.component.lesson_detail', compact('lesson', 'course'));
     }
+
+    public function storeCourseComment(StoreCommentRequest $request, Course $course)
+    {
+        $course->comments()->create([
+            'content' => $request->content, 
+            'user_id' => Auth::user()->id, 
+            'rate' => config('rate.default'),
+            ]);
```
Comment (huyenttm-0901) [inline app/Http/Controllers/User/StudentController.php]:
```suggestion
            'rate' => config('rate.default'),
        ]);
```

File: resources/views/layouts/menu_user.blade.php [hunk 2]
```diff
@@ -15,7 +15,11 @@
         @auth
             <div class="info-container">
                 <div class="avatar">
-                    <img src="{{ asset(config('title.avatar_default')) }}" alt="logo">
+                    @if (Auth::user()->image)
+                        <img src="{{ asset(config('img.img_path'). Auth::user()->image->url) }}" alt="">
```
Comment (huyenttm-0901) [inline resources/views/layouts/menu_user.blade.php]:
```suggestion
                        <img src="{{ asset(config('img.img_path') . Auth::user()->image->url) }}" alt="">
```

File: resources/views/user/component/lessons.blade.php [hunk 3]
```diff
@@ -52,25 +52,29 @@
...
                             </div>
                             <div class="comment-submit">
-                                <button>@lang('label.comment_button')</button>
+                                <button type="submit" form="comment-form">@lang('label.comment_button')</button>
                             </div>
                         </div>
                     @endauth
-                    <div class="comment-detail-student">
-                        @foreach ($course->comments as $key => $comment)
+                    @foreach ($course->comments as $key => $comment)
```
Comment (huyenttm-0901) [inline resources/views/user/component/lessons.blade.php]: $key ko cần thì bỏ đi em

**Comment by wowKimsohyun17** [comment]
ready

**Pull request merged**